Vite 5.0 Support

```
$ npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: @vitejs/plugin-vue2@2.2.0
npm ERR! Found: vite@5.0.0-beta.4
npm ERR! node_modules/vite
npm ERR!   dev vite@"^5.0.0-beta.4" from the root project
```

--- a/package.json
+++ b/package.json
@@ -36,7 +36,7 @@
   },
   "homepage": "https://github.com/vitejs/vite-plugin-vue2/#readme",
   "peerDependencies": {
-    "vite": "^3.0.0 || ^4.0.0",
+    "vite": "^3.0.0 || ^4.0.0 || ^5.0.0",
     "vue": "^2.7.0-0"
   },
   "devDependencies": {

